qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -198,12 +200,12 @@ def test_snap_schedule_list_post_schedule_remove(self):
         """Test listing snap schedules post removal of a schedule"""
         self.mount_a.run_shell(['mkdir', '-p', TestSnapSchedules.TEST_DIRECTORY])
 
-        self.fs_snap_schedule_cmd('add', f'path={TestSnapSchedules.TEST_DIRECTORY}', 'snap-schedule=1h')
+        self.fs_snap_schedule_cmd('add', path=f'{TestSnapSchedules.TEST_DIRECTORY}', **{'snap-schedule':'1h'})
```

Comment:
nit, better off putting `snap_schedule` as plain kwarg. like
```py
snap_schedule='1M')
```
so it is more consistent and more readable. the signature including `snap-schedule` is defined at https://github.com/ceph/ceph/blob/6824b29ca1d843cbac81ca2d7caef14640b21826/src/pybind/mgr/snap_schedule/module.py#L115-L121

and is normalized at https://github.com/ceph/ceph/blob/6824b29ca1d843cbac81ca2d7caef14640b21826/src/pybind/mgr/mgr_module.py#L382.

Reply:
Thanks for the suggestion! Fixed.